tests/unit/test_cv_container_tools.py

```diff
@@ -6,18 +6,25 @@
 import pprint
 from ansible_collections.arista.cvp.plugins.module_utils.container_tools import CvContainerTools
 from tests.lib import mock
-from tests.unit import data
+from tests.data import container_tools_unit
 
 LOGGER = logging.getLogger(__name__)
 
 
+CVP_DATA_CONTAINERS_INIT = {
+    'Tenant': {
+        'name': 'Tenant',
+        'key': 'root',
+        'parentContainerId': None}
+}
+
 # ---------------------------------------------------------------------------- #
 #   FIXTURES
 # ---------------------------------------------------------------------------- #
 
 @pytest.fixture
 def cvp_database():
-    database = mock.MockCVPDatabase()
+    database = mock.MockCVPDatabase(containers=CVP_DATA_CONTAINERS_INIT)
```

Comment:
In the last commits of #434, the mock database is always initialised with the root container. The data needed for the test is imported in the mock database as an indirect pytest parameter. So either in this PR or the other there will be conflicts.